src/testStatusCodeLens.ts

```diff
@@ -0,0 +1,25 @@
+"use strict";
+import { CodeLens, Range } from "vscode";
+
+export class TestStatusCodeLens extends CodeLens {
+    public static parseOutcome(outcome: string): string {
+        if (outcome === "Passed") {
+            return "✔️";
```

Comment:
Test on macOS, it is showing a black check mark (https://emojipedia.org/heavy-check-mark/) which is hard to see. 
Two improvements:

- Using another better check mark: https://emojipedia.org/white-heavy-check-mark/. Or you may have a better one? 😄 
- Using the unicode, it would be good to maintain 
![image](https://user-images.githubusercontent.com/1050213/33176645-bce5b67a-d09a-11e7-85b3-03c3529672fd.png)

Reply:
I've added settings now so the user can override what we display.

I've kept the heavy check mark for platforms apart from OS X, in which case it uses the white heavy check mark (I'm not a big fan of it, but I can't find any alternatives).

Since the user can specify the text, they could even switch to "passing", "FAILING" etc if they want so hopefully this is OK?

Reply:
Looks great to me!